geozero/Cargo.toml

```diff
@@ -34,6 +36,7 @@ geos = { version = "8.0", optional = true }
 gdal = { version = "0.12", default-features = false, optional = true }
 gdal-sys = { version = "0.6", optional = true }
 lyon = { version = "0.17", optional = true }
+log = "0.4.17"
```

Comment:
Note that I added the log crate. I'm only using it in one place at the moment, but a lot of our error messages are kind of opaque, and logging is an easy way to add context about line number, etc.